packages/app/app/components/Playlists/index.tsx

```diff
@@ -23,9 +23,10 @@ const EmptyState = () => {
 type PlaylistsProps = {
   playlists: Playlist[];
   handleImportFromFile: React.MouseEventHandler;
+  createNew: (name: any) => void;
```

Comment:
Use a correct type for this (don't use `any`)

Reply:
there are no type in action, but ok

Reply:
fixed